cmd/rmr.go

```diff
@@ -59,10 +58,6 @@ func openController(dpath string) (*os.File, error) {
 
 func rmr(ctx *cli.Context) error {
 	setup(ctx, 1)
-	if runtime.GOOS == "windows" {
```

Comment:
I think the current implementation is better, because we have not find a way to communicate with running client, could be fixed in future.